### DISCUSSION TITLE
Bulk edit date

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#949] [Enhancements]
[Roadmap 26 Votes](https://roadmap.ente.io/bulk-edit-exif-date-p-3515/)

>  Love the new photo date editing feature, but would be even better if we could do a bulk edit. Would also be ideal to be able to change just the date and not the times of the photos selected. 

> I also think bulk editing date information would be incredibly useful. I just uploaded a whole folder of photos from a specific date, but the original exif data was incorrect, so I had to edit the dates on each photo manually to get them to show up in my ente timeline correctly 

Related
# https://github.com/ente-io/photos-app/issues/1389

**Comment by haritvsmurali** [comment]
+1

**Comment by laurenspriem** [comment]
Included in the latest v1 release!
